tests/10apidoc/09synced.pl

```diff
@@ -205,7 +205,34 @@ sub await_sync_presence_contains {
 }
 
 
-push @EXPORT, qw( assert_room_members assert_state_room_members_matches );
+push @EXPORT, qw( assert_room_members assert_state_room_members_matches assert_room_state);
+
+# assert that the state body of a sync response is made up of the given state types.
```

Comment:
nice to have: generally we seem to be using POD for this sort of thing. It'd be nice to be consistent, even if it's slightly more verbose.